ppb/events.py

```diff
@@ -87,6 +83,12 @@ def __event__(self, bag, fire_event):
 
 # Remember to define scene at the end so the pargs version of __init__() still works
 
+from ppb.scenes import BaseScene
```

Comment:
Why are you moving the imports from the top of the file?

Reply:
Removing the ABC means we need another type for the type hints, and BaseScene depends on `ppb.events` which now depends on `ppb.scenes`. So a known trick for allowing circular imports.

Long term, I think we need to move `EventMixin` or the event definitions elsewhere so we're not doing this if we can avoid it.

Reply:
:+1: on breaking the circular dependency.